config/orm.xml

```diff
@@ -33,8 +33,8 @@
         <parameter key="doctrine.orm.metadata.annotation.class">Doctrine\ORM\Mapping\Driver\AnnotationDriver</parameter>
         <parameter key="doctrine.orm.metadata.xml.class">Doctrine\ORM\Mapping\Driver\SimplifiedXmlDriver</parameter>
         <parameter key="doctrine.orm.metadata.yml.class">Doctrine\ORM\Mapping\Driver\SimplifiedYamlDriver</parameter>
-        <parameter key="doctrine.orm.metadata.php.class">Doctrine\ORM\Mapping\Driver\PHPDriver</parameter>
-        <parameter key="doctrine.orm.metadata.staticphp.class">Doctrine\ORM\Mapping\Driver\StaticPHPDriver</parameter>
+        <parameter key="doctrine.orm.metadata.php.class">Doctrine\ORM\Persistence\Driver\PHPDriver</parameter>
```

Comment:
Shouldn't this be

```suggestion
        <parameter key="doctrine.orm.metadata.php.class">Doctrine\Persistence\Mapping\Driver\PHPDriver</parameter>
```

?

Reply:
It seems correct according to the upgrade documentation https://github.com/doctrine/orm/blob/3.1.x/UPGRADE.md#metadata-drivers

Reply:
Actually you're right I made a typo nice catch!